tests/compilers/glimmer-compiler-test.js

```diff
@@ -13,6 +13,110 @@ module('glimmer-template-precompiler', function(hooks) {
     input.dispose();
   });
 
+  module('pragma tag extraction', () => {
+    let templateVariations = [
+      '\n{{use-component-manager "very-custom-manager"}}\n\n<nyan-cat>wot up</nyan-cat>',
+      '{{use-component-manager "very-custom-manager"}}\n\n<nyan-cat>wot up</nyan-cat>',
+      '{{ use-component-manager "very-custom-manager"}}\n\n<nyan-cat>wot up</nyan-cat>',
+      '{{ use-component-manager "very-custom-manager" }}\n\n<nyan-cat>wot up</nyan-cat>',
+      '{{\nuse-component-manager "very-custom-manager"}}\n\n<nyan-cat>wot up</nyan-cat>',
+      '{{use-component-manager "very-custom-manager"\n}}\n\n<nyan-cat>wot up</nyan-cat>',
+      '{{\nuse-component-manager "very-custom-manager"\n}}\n\n<nyan-cat>wot up</nyan-cat>',
+      '{{\nuse-component-manager\n"very-custom-manager"\n}}\n\n<nyan-cat>wot up</nyan-cat>',
+      '\n{{use-component-manager "very-custom-manager"}}\n\n<nyan-cat>wot up</nyan-cat>',
+      `\n{{use-component-manager 'very-custom-manager'}}\n\n<nyan-cat>wot up</nyan-cat>`
+    ];
+    let extractedPragmaTemplateVariations = [
```

Comment:
this is a bit brittle but I wanted to make sure extraction works again many scenarios; firstly, I had 9 variables which didn't seem better than having an array.